### ISSUE TITLE
onProgressUpdate progress value is incorrect

### ISSUE THREAD

**Issue by flickstree** [nichucs/HLS2MP4#5]
onProgressUpdate progress value is incorrect or not getting success status. Can you please add onSuccess in DownloadListener listener interface and can you add cancel download function  to cancel the on going download task

**Comment by devt2328** [comment]
> onProgressUpdate progress value is incorrect or not getting success status. Can you please add onSuccess in DownloadListener listener interface and can you add cancel download function to cancel the on going download task

Yes right. I had faced same problem. So please give correct value with success status and also provide some interfaces to handle some scenarios like cancel.

**Comment by sumitflickstree** [comment]
any updates?

**Comment by nichucs** [comment]
This repo is not maintained any kore. Please feel free to raise a pull request if you have something good in hand. Thanks.

Sent from Yahoo Mail on Android 
 
  On Wed, 17 May 2023 at 4:25 pm, ***@***.***> wrote:   


any updates?
